options/m_option.c

```diff
@@ -482,7 +482,7 @@ const m_option_type_t m_option_type_int64 = {
     .equal = int64_equal,
 };
 
-static int parse_byte_size(struct mp_log *log, const m_option_t *opt,
+int parse_byte_size(struct mp_log *log, const m_option_t *opt,
```

Comment:
Better define a dummy m_option (on the stack at your call site) and call the normal parser wrapper, instead of making this function non-static.

Reply:
> normal parser wrapper

I'm not sure what is this referring to.

Reply:
`m_option_parse()`

Reply:
Yeah. I realized that after writing that comment, sorry.

Reply:
Addressed.